fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -527,7 +538,7 @@ public Response createOrReplaceObjectRdf(
                     }
 
                     doInDbTx(() -> {
-                        if (resourceExists) {
+                        if (resourceExists && !(resource() instanceof Tombstone)) {
```

Comment:
I feel like I was going to use `isTombstoneOverwrite` here. Not sure why I didn't, we know it should have been checked because both places also rely on `resourceExists` to be true. IDK I might look into this a bit more and make a new ticket/PR.